README.md

```diff
@@ -1 +1 @@
-review 
\ No newline at end of file
+hello microverse project 
```

Comment:
- [ ] Add a descriptive README to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.